03_launch_mgmt_cluster.sh

```diff
@@ -225,25 +184,18 @@ EOF
   else
     # Deploy Ironic using deploy.sh script
 
-    # Update container images to use local ones
-    cp "${BMOPATH}/ironic-deployment/ironic/ironic.yaml" "${BMOPATH}/ironic-deployment/ironic/ironic.yaml.orig"
-    cp "${BMOPATH}/ironic-deployment/keepalived/keepalived_patch.yaml" "${BMOPATH}/ironic-deployment/keepalived/keepalived_patch.yaml.orig"
-    update_kustomization_images "${BMOPATH}/ironic-deployment/ironic/ironic.yaml"
-    update_kustomization_images "${BMOPATH}/ironic-deployment/keepalived/keepalived_patch.yaml"
+    update_component_image Ironic "${IRONIC_LOCAL_IMAGE:-$IRONIC_IMAGE}"
+    update_component_image Mariadb "${MARIADB_LOCAL_IMAGE:-$MARIADB_IMAGE}"
+    update_component_image Keepalived "${IRONIC_KEEPALIVED_LOCAL_IMAGE:-$IRONIC_KEEPALIVED_IMAGE}"
+    update_component_image IPA-downloader "${IPA_DOWNLOADER_LOCAL_IMAGE:-$IPA_DOWNLOADER_IMAGE}"
```

Comment:
I dont think assigning the vars in a function call is a good idea. Would you please fix the value of the image before calling the fucntion.

Reply:
I'm not sure I understand. Something like this?
```bash
TMP_IMAGE="${IRONIC_LOCAL_IMAGE:-$IRONIC_IMAGE}"
update_component_image Ironic "${TMP_IMAGE}"
```
Or maybe with if/else?
```bash
if [ -n "${CAPM3_LOCAL_IMAGE}" ]; then
  update_component_image CAPM3 "${CAPM3_LOCAL_IMAGE}"
else
  update_component_image CAPM3 "${CAPM3_IMAGE}"
fi
```
I actually had this if/else solution first but thought it was a bit cluttered. But I can put it back if you think it is better?

Reply:
For the sake of clarity yes please.

Reply:
Done. Note that the files needed to backup/restore changed since we use the new make targets in BMO now. I think it also shows why it would be good to change this: it is a dependency between metal3-dev-env and BMO/CAPM3. I'll create a follow up PR for fixing this.

Reply:
Didn't get the later part of the comment. Can you elaborate?

Reply:
Basically we edit files in BMO/CAPM3 from metal3-dev-env, so there is a direct dependency between them. Like if we for example move or rename one of the deployment files in BMO, it would break these scripts since they depend on those files directly. Of course there will always be some relation between them, like using make targets, but this is a bit too much in my opinion and I have some ideas for how to reduce it.

Reply:
Allright, not sure if I got it fully but perhaps once you have a PR, it will be more visible . Good work in this one.